core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -882,6 +882,16 @@ object LogManager {
   val LogStartOffsetCheckpointFile = "log-start-offset-checkpoint"
   val ProducerIdExpirationCheckIntervalMs = 10 * 60 * 1000
 
+  val LogCleanerReconfigurableConfigs = Set(
```

Comment:
Seems like it might be more natural to put this in the `LogCleaner` companion object.